### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by fcami**
Add tests for JH and ODS Dashboard, plus running simple notebooks.

### CODE REVIEW

File: tests/Tests/400_ods_dashboard/400_ods_dashboard_auth.robot [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Accelerated Setup Suite
+  Set Library Search Order  SeleniumLibrary
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+
+*** Variables ***
+
+*** Test Cases ***
+
+Open RHODS Dashboard
+  [Tags]  Start Sequence
```
Comment (bdattoma) [inline tests/Tests/400_ods_dashboard/400_ods_dashboard_auth.robot:23]: what's this tag for?

File: tests/Tests/500__jupyterhub/jupyterhub-basic-auth.robot [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  Capture Page Screenshot
+
+Can Login to Jupyterhub
+  [Tags]  Start Sequence
+  Capture Page Screenshot
+  Login To Jupyterhub  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  ${authorization_required} =  Is Service Account Authorization Required
+  Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+  Wait Until Page Contains Element  xpath://span[@id='jupyterhub-logo']
+  Capture Page Screenshot
```
Comment (bdattoma) [inline tests/Tests/500__jupyterhub/jupyterhub-basic-auth.robot:44]: why do you need these test cases in a separated file? we already have same ones in the repository. Before duplicating them we should understand the reason behind and if there are alternatives. 

For example, if you need those test cases for perf testing, then we can add a specific tag to the already implemented ones and assess the possibility to do changes according your needs

File: tests/Tests/400_ods_dashboard/400_ods_dashboard_auth.robot (hunk 1 above)
Comment (fcami) [reply tests/Tests/400_ods_dashboard/400_ods_dashboard_auth.robot:23]: We've identified dashboard logon to be problematic at scale. We need to be able to test just the logon itself.

File: tests/Tests/500__jupyterhub/jupyterhub-basic-auth.robot (hunk 2 above)
Comment (fcami) [reply tests/Tests/500__jupyterhub/jupyterhub-basic-auth.robot:44]: I'm afraid this was written a long time ago by Erwan. What we're trying to do is to stimulate a stampede at different stages:
* (1) being able to logon to the dashboard at scale
* (2): (1)+being able to logon to JH at scale
* (3): (2)+being able to launch the same notebook at scale
Please let me know if this is already achievable.

Comment (kpouget) [reply tests/Tests/500__jupyterhub/jupyterhub-basic-auth.robot:44]: After discussions with @tarukumar I found out a simpler way to do this, without having to split the test into multiple files, with the `--exclude` flag.

See this file:
https://github.com/openshift-psap/ods-ci/blob/a3c68899045958d29bfc32407a199573eaa8b9d9/tests/Tests/500__jupyterhub/test-jupyterlab-psap-simplenotebook.robot

which defiles the following tags: `RHODS`, `Jupyterhub` and `Notebook`.
Our idea is to study the large scale behavior with `--exclude JupyterhubANDNotebook`, or `--exclude Notebook` or (nothing excluded)

**Comment by kpouget** [comment]
we'll update the PR when François is back from PTO (next week), but meanwhile if you have any feedback on the content of this (single) file:
 https://github.com/openshift-psap/ods-ci/blob/a3c68899045958d29bfc32407a199573eaa8b9d9/tests/Tests/500__jupyterhub/test-jupyterlab-psap-simplenotebook.robot

This test runs successfully from my laptop, we'll run it in the CI with 10 and more users simultaneously as soon as possible

```
==============================================================================
Test-Jupyterlab-Psap-Simplenotebook
==============================================================================
Open RHODS Dashboard                                                  | PASS |
------------------------------------------------------------------------------
Can Launch Jupyterhub                                                 | PASS |
------------------------------------------------------------------------------
Can Login to Jupyterhub                                               | PASS |
------------------------------------------------------------------------------
Can Spawn Notebook                                                    | PASS |
------------------------------------------------------------------------------
Git Clone the PSAP notebooks                                          | PASS |
------------------------------------------------------------------------------
Run the PSAP jh-at-scale Simple Notebook                              | PASS |
------------------------------------------------------------------------------
Clean up the files and folders we created                             | PASS |
------------------------------------------------------------------------------
Can Close Notebook when done                                          | PASS |
------------------------------------------------------------------------------
Test-Jupyterlab-Psap-Simplenotebook                                   | PASS |
8 tests, 8 passed, 0 failed
==============================================================================
```

**Comment by bdattoma** [comment]
@kpouget I'm closing this PR because I assume you got this covered or is not needed anymore, since there were no more progress on it. Feel free to re-open or start a new PR

**Pull request closed without merging**